src/libsystemd/meson.build

```diff
@@ -70,6 +70,12 @@ libsystemd_sources = files('''
         sd-hwdb/hwdb-util.c
         sd-hwdb/hwdb-util.h
         sd-hwdb/sd-hwdb.c
+        sd-lldp/sd-lldp.c
+        sd-lldp/lldp-internal.h
+        sd-lldp/lldp-neighbor.c
+        sd-lldp/lldp-neighbor.h
+        sd-lldp/lldp-network.c
+        sd-lldp/lldp-network.h
```

Comment:
I don't think we want to move `lldp` to be part of systemd. I know that system-network is not exposed as a public library. Most likely we should start doing that.